examples/conditionals-while/src/main/java/io/littlehorse/examples/ConditionalsWhileExample.java

```diff
@@ -37,10 +37,10 @@ public static Workflow getWorkflow() {
                 );
 
                 wf.doWhile(
-                    wf.condition(numDonuts, Comparator.GREATER_THAN, 0),
+                        wf.condition(numDonuts, Comparator.GREATER_THAN, 0),
                     handler -> {
-                        handler.execute("eating-donut", numDonuts);
                         wf.mutate(numDonuts, VariableMutationType.SUBTRACT, 1);
```

Comment:
:porg: why is it `wf.mutate` and not `handler.mutate`? I know that's been there since before this PR but maybe we should fix it :smile: